README.md

```diff
@@ -1,14 +1,15 @@
-[![CI Status](https://github.com/se-edu/addressbook-level3/workflows/Java%20CI/badge.svg)](https://github.com/se-edu/addressbook-level3/actions)
+[![CI Status](https://github.com/AY2324S1-CS2103T-W17-2/tp/workflows/Java%20CI/badge.svg)](https://github.com/AY2324S1-CS2103T-W17-2/tp/actions)
 
 ![Ui](docs/images/Ui.png)
 
-* This is **a sample project for Software Engineering (SE) students**.<br>
+* LetsGetHired is **team W17-2's CS2103T Project for AY23/24**.<br>
   Example usages:
-  * as a starting point of a course project (as opposed to writing everything from scratch)
-  * as a case study
-* The project simulates an ongoing software project for a desktop application (called _AddressBook_) used for managing contact details.
-  * It is **written in OOP fashion**. It provides a **reasonably well-written** code base **bigger** (around 6 KLoC) than what students usually write in beginner-level SE modules, without being overwhelmingly big.
-  * It comes with a **reasonable level of user and developer documentation**.
-* It is named `AddressBook Level 3` (`AB3` for short) because it was initially created as a part of a series of `AddressBook` projects (`Level 1`, `Level 2`, `Level 3` ...).
-* For the detailed documentation of this project, see the **[Address Book Product Website](https://se-education.org/addressbook-level3)**.
-* This project is a **part of the se-education.org** initiative. If you would like to contribute code to this project, see [se-education.org](https://se-education.org#https://se-education.org/#contributing) for more info.
+  * Track job applications
+  * Overview of pending, rejected and accepted job offers you have
+  * All so you will never miss interviews or offer acceptance deadlines!
+
+* As a group of Year 2 Computing Students, school takes too much of our attention and there is a possibility of 
+  forgetting about job application follow-ups and interviews that we may have.
+* This app is designed to help students to track their job applications amidst the heavy job-hunting season.
+* For the detailed documentation of this project, read our **[User Guide](docs/UserGuide.md)**.
```

Comment:
Nice ! Can you add the links to the Developer Guide, About us, and Contact us (Not really sure abt this one yet) ? Can refer to the original AB3 readme if unsure how to structure it but I think this is gd too! Keep it up